src/PythonClient/src/quixstreams/models/streamconsumer/streameventsconsumer.py

```diff
@@ -46,14 +48,22 @@ def _finalizerfunc(self):
     @property
     def on_data_received(self) -> Callable[['StreamConsumer', EventData], None]:
         """
-        Gets the handler for when the stream receives event. First parameter is the stream the event is received for, second is the event.
+        Gets the handler for when an events data package is received for the stream.
+
+        Returns:
+            Callable[['StreamConsumer', EventData]]:
```

Comment:
Not sure if this is consistent with other places (files I already revieved).

> Callable[['StreamConsumer', EventData], None]

means it takes two parameters and returns type "None" (c# void), so leaving the return type off might not be the best.